WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -429,10 +429,6 @@ - (void)viewDidAppear:(BOOL)animated
         [self scrollToElement:QuickStartTourElementViewSite];
         self.shouldScrollToViewSite = NO;
     }
-    if (![Feature enabled:FeatureFlagNewNavBarAppearance] &&
```

Comment:
@Gio2018 - This modification looks odd, but it seems to be logically correct.  Do you know if this is right, or there's a bug in here?